src/Serilog.AspNetCore/SerilogWebHostBuilderExtensions.cs

```diff
@@ -41,5 +41,25 @@ public static IWebHostBuilder UseSerilog(this IWebHostBuilder builder, Serilog.I
                 collection.AddSingleton<ILoggerFactory>(new SerilogLoggerFactory(logger, dispose)));
             return builder;
         }
+
+        /// <summary>Sets Serilog as the logging provider.</summary>
+        /// <param name="builder">The web host builder to configure.</param>
+        /// <param name="configureSerilog">The delegate for configuring the <see cref="LoggerConfiguration" /> that will be used to construct a <see cref="Logger" />.</param>
+        /// <returns>The web host builder.</returns>
+        public static IWebHostBuilder UseSerilog(this IWebHostBuilder builder, Action<WebHostBuilderContext, LoggerConfiguration> configureSerilog)
```

Comment:
Since the action applies to a logger configuration, can we call the parameter `configureLogger` instead of `configureSerilog`?